modules/ROOT/pages/ts-no-T1-acks.adoc

```diff
@@ -0,0 +1,32 @@
+= Missing TA1 acknowledgments for my transmissions
+
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+
+Anypoint Partner Manager sends TA1 acknowledgments back for any received transmission, if configured to do so. Your trading partner may contact you to tell you that they are not receiving TA1 acknowledgements.
+
+== Causes
+
+Any of the following can lead to this type of error:
+
+. Your trading partners X12 settings are incorrectly configured do not send TA1’s back to the partner.
```

Comment:
Grammatically incorrect. Please reread for accuracy:
```suggestion
. Your trading partners' X12 settings are incorrectly configured to not return TA1s.
```

Reply:
F